osidb/helpers.py

```diff
@@ -238,3 +240,98 @@ def safe_get_response_content(response: Response):
         return response.json()
     except JSONDecodeError:
         return response.text
+
+
+class JSONSocketHandler(logging.handlers.SocketHandler):
+    def __init__(
+        self,
+        *args,
+        logfile: str,
+        # fallback_logfile: Optional[str] = None,
+        # retries: int = 2,
+        # delay: int = 1,
+        **kwargs,
+    ) -> None:
+        super().__init__(*args, **kwargs)
+        self.logfile = logfile
+        # self.retries = retries
+        # self.delay = delay
+
+        # self.fallback_handler = None
+        # if fallback_logfile is not None:
+        #     self.fallback_handler = logging.FileHandler(filename=fallback_logfile)
+
+    def makePickle(self, record: logging.LogRecord) -> bytes:
```

Comment:
For the method names we use the snake-case instead of the camel-case so I would be for preserving the consistency and got with the camel-case here too.

Reply:
I see below that the parent class uses the camel-case too so you probably preserved that. Then, I have no strong preference. The mixture seems to be unavoidable.

Reply:
Yes, this done in order to override the method which serializes the data in the handler and unfortunately they are using camel-case :/